### ISSUE TITLE
Multi-window apps stutter if single window hidden.

### ISSUE THREAD

**Issue by bisby** [swaywm/sway#7987]
### Please fill out the following:
- **Sway Version:**
  - `swaymsg -t get_version` : `sway version 1.8.1`

- **Debug Log:**
  - `sway -c /etc/sway/config -d 2> ~/sway.log`
  - https://gist.github.com/bisby/8e2dea5e8c772c4be093c5004c8cd31a
  - In this log I: open zoom. join a meeting. open chat. move chat to a tabbed pane behind a terminal. Fullscreen my video feed. end the meeting.

- **Configuration File:**
Reproduced using the stock config `/etc/sway/config` when generating the debug log.

- **Description:**
  - The steps you took in plain English to reproduce the problem.

Multi-window apps (Zoom, Steam) becomes laggy when ONE of the windows is not visible.

If I have chat and the video call both open and visible, everything runs fine. If chat gets moved to a non-visible tab, the video starts becoming laggy.

It seems like "hidden applications" logic is aggressively interfering with rendering ANY of the application if only a part of it is hidden.

* Open Zoom call, open chat
* Move chat to a non-visible tab
* Zoom video call starts stuttering
* Move chat back to visible
* Zoom clears up and runs smoothly.

Alternatively
* Open Zoom call in dual monitor mode (this opens 1 participant window and 1 current speaker window)
* Fullscreen one of the windows
* The window fullscreens but video feeds stutter
* Unfullscreen the window
* Video feeds return to normal.

Similarly, the Steam Overlay in games will often stutter when opened, if Steam has a non visible window open.

"Stutter" in these contexts is rendering the window at roughly 1fps.

**Comment by bblacher** [comment]
I'm also experiencing this with Steam, I thought it was a Steam bug though. Could also be a xwayland problem (?)

**Comment by emersion** [comment]
Duplicate of https://github.com/swaywm/sway/issues/7557
